front/app/containers/Admin/initiatives/index.tsx

```diff
@@ -4,44 +4,45 @@ import { Outlet as RouterOutlet } from 'react-router-dom';
 
 // components
 import HelmetIntl from 'components/HelmetIntl';
-import TabbedResource from 'components/admin/TabbedResource';
-import Button from 'components/UI/Button';
+import NavigationTabs, {
+  Tab,
+  TabsPageLayout,
+} from 'components/admin/NavigationTabs';
+import Link from 'utils/cl-router/Link';
 
 // i18n
 import messages from './messages';
 import { WrappedComponentProps } from 'react-intl';
 import { injectIntl } from 'utils/cl-intl';
 
-// tracks
-import { trackEventByName } from 'utils/analytics';
-import tracks from './tracks';
-
 // styles
 import { InsertConfigurationOptions, ITab } from 'typings';
 import Outlet from 'components/Outlet';
 import { insertConfiguration } from 'utils/moduleUtils';
+import styled from 'styled-components';
+
+// utils
+import { matchPathToUrl } from 'utils/helperUtils';
+
+const StyledTabsPageLayout = styled(TabsPageLayout)`
+  padding-left: 44px;
+`;
 
 const InitiativesPage = memo<WrappedComponentProps & WithRouterProps>(
   ({ intl: { formatMessage }, location }) => {
     const [tabs, setTabs] = useState<ITab[]>([
+      {
+        label: formatMessage(messages.proposals),
+        name: 'proposals',
+        url: '/admin/initiatives/proposals',
```

Comment:
"Proposals" is the new (relatively) way to call "Initiatives". So, maybe it makes sense to remove this duplication and leave it short `/admin/initiatives` or old `/admin/initiatives/manage`?